site/content/docs/5.3/dark-mode.md

```diff
@@ -644,6 +644,223 @@ Additional variables for dark-mode (temporary)
+        <use xlink:href="/docs/5.3/assets/img/boosted-sprite.svg#buy"/>
+      </svg>
+      Dismissible tag
+      <button class="close" aria-labelledby="labelTag22"><span class="visually-hidden">Close</span></button>
+    </span></li>
+    <li>
+      <p><span class="tag disabled" id="labelTag52" aria-disabled="true">
+        <svg fill="currentColor" width="1.5rem" height="1.5rem" aria-hidden="true" focusable="false">
+          <use xlink:href="/docs/5.3/assets/img/boosted-sprite.svg#buy"/>
+        </svg>
+        Input
+        <button class="close" aria-labelledby="labelTag52" disabled><span class="visually-hidden">Close</span></button>
+      </span></p>
+    </li>
+  </ul>
+</div>
+
+<h4 class="mt-3">Light theme on container</h4>
+
+<div class="border border-tertiary p-3 bg-body" data-bs-theme="light">
+  <ul class="list-unstyled d-flex gap-2 flex-wrap m-0">
+    <li>
+      <input type="checkbox" class="btn-check" id="btncheck-mobile3" autocomplete="off">
+      <label class="tag" for="btncheck-mobile3"><span class="visually-hidden">Filter by</span>Mobile</label>
+    </li>
+    <li>
+      <input type="checkbox" class="btn-check" id="btncheck-tv3" autocomplete="off" checked>
+      <label class="tag" for="btncheck-tv3">
+        <svg width="1.5rem" height="1.5rem" viewBox="0 0 1000 1000" aria-hidden="true" focusable="false">
+          <path fill="currentColor" d="M75,200V720H225v80H775V720H925V200H75ZM500,755a30,30,0,1,1,30-30A30,30,0,0,1,500,755Zm365-95H135V260H865V660Z"></path>
+        </svg>
+        <span class="visually-hidden">Filter by</span>TV
+      </label>
+    </li>
+    <li><a class="tag" href="#">1. Introduction</a></li>
+    <li>
+      <span class="tag" id="labelTag13">
+        Dismissible tag
+        <button class="close" aria-labelledby="labelTag13"><span class="visually-hidden">Close</span></button>
+      </span>
+    </li>
+    <li><span class="tag tag-sm" id="labelTag23">
+      <svg width="1rem" height="1rem" fill="currentColor" aria-hidden="true" focusable="false">
+        <use xlink:href="/docs/5.3/assets/img/boosted-sprite.svg#buy"/>
+      </svg>
+      Dismissible tag
+      <button class="close" aria-labelledby="labelTag23"><span class="visually-hidden">Close</span></button>
+    </span></li>
+    <li>
+      <p><span class="tag disabled" id="labelTag53" aria-disabled="true">
+        <svg fill="currentColor" width="1.5rem" height="1.5rem" aria-hidden="true" focusable="false">
+          <use xlink:href="/docs/5.3/assets/img/boosted-sprite.svg#buy"/>
+        </svg>
+        Input
+        <button class="close" aria-labelledby="labelTag53" disabled><span class="visually-hidden">Close</span></button>
+      </span></p>
+    </li>
+  </ul>
+</div>
+
+<h4 class="mt-3">Dark theme on component</h4>
+
+<div class="border border-tertiary p-3" style="background-color: #282d55">
+  <ul class="list-unstyled d-flex gap-2 flex-wrap m-0" data-bs-theme="dark">
```

Comment:
As discussed together, just raising a question here: Should the `data-bs-theme` sit on the list or on each `.tag` item?

My 2cents here are that it should be on `.tag` directly to test it well.